### ISSUE TITLE
Greenbelt Map!

### ISSUE THREAD

**Issue by amaliebarras** [open-austin/project-ideas#71]
### What problem are we trying to solve?

Exploring Austin's greenbelt seems to rely heavily on word-of-mouth, there is no collaborative map solution.

### Who will benefit (directly and indirectly) from this project?

People visiting Austin who may not know who to ask about greenbelt entrances, people who want to search for map spots based on activities

### [greenbelt map](https://goo.gl/maps/LraKQQrSuED2)

Some pals of mine have started collecting data in a google map form. This map is open for anyone to edit! 

### What are the next steps (validation, research, coding, design)? 

Technically, google maps seems like an OK platform for this, but eventually, a more intuitive pin-dropping tool where you can add data anonymously and not have to authenticate through google, may be better!

### What help is needed at this time?

Project Leader: someone to make a slack channel, round up troops, and make sure data is recorded
Data Collection: folks who want to explore the greenbelt and record new spots and activities (cliff jumping, rope swinging, kayak friendly, etc)

Lastly, the data is here if anyone wants to do something else with it!

**Comment by amaliebarras** [comment]
Thought of @mapineda :)

**Comment by mapineda** [comment]
@amaliebarras this is awesome. Let me close the recycling-please and xeritopia loop and I'll get on this as soon as possible.

**Comment by amaliebarras** [comment]
I totally hear that #somanyprojects #solittletime

**Comment by mapineda** [comment]
Would be cool to do a "Greenbelt GO AR app"

**Comment by werdnanoslen** [comment]
I made an entirely javascript boilerplate app for mobile map i/o here: [https://github.com/werdnanoslen/mobile-map-io](https://github.com/werdnanoslen/mobile-map-io), if that might be a good place to start from.

**Comment by edsfocci** [comment]
@amaliebarras about Greenbelt Map.kml.zip, where did you get the data from? Can we talk to someone from Pars & Recreation, or similar organization?

I found this:
https://tpwd.texas.gov/fishboat/fish/recreational/lakes/greenbelt/

**Comment by luqmaan** [comment]
@werdnanoslen github.com/open-austin/austingreenmap

**Comment by edsfocci** [comment]
Disregard my link, it it the wrong part of Texas. However, some way of figuring out the official boundaries of the Greenbelt may be helpful.

**Comment by edsfocci** [comment]
Found this map:
https://data.austintexas.gov/dataset/City-Of-Austin-Parks/99qw-4ixs

**Comment by amaliebarras** [comment]
Hey @edsfocci ! Some friends of mine went out into the field and collected the data over the course of a couple weeks. 

Let's keep track of parks data requests! I think there is certainly parks data that isn't out in the open. I bet between Open Austin and my buddies on the outside, there are people who'd be into some in the field data exploration :) 

Also, going to add do your list of links here: 
ftp://ftp.ci.austin.tx.us/GIS-Data/Regional/coa_gis.html Map downloads?
http://www.austintexas.gov/department/slacklining - It's just a list of parks instead of a map :'(

**Comment by Rmcavin** [comment]
Is this project still lacking leadership/data/development? I'm a GIS specialist who has recently started GIS Dev as a hobby. I'd be happy to help with this project, seems awesome :)

**Comment by werdnanoslen** [comment]
@amaliebarras ☝️

**Comment by amaliebarras** [comment]
@rmcavin go for it! I haven't had time to carry this forward but if you'd like to adopt it I'm happy to transfer all the knowledge I have.

**Comment by Rmcavin** [comment]
That would be great :)

**Comment by werdnanoslen** [comment]
Yay! @Rmcavin could you please go to [this page](https://github.com/orgs/open-austin/teams) and request to join the "all" group so i can make you official?

**Comment by Rmcavin** [comment]
@werdnanoslen I looked around the page and didn't see any join group options :/ Not sure if I'm overlooking something lol.

**Comment by amaliebarras** [comment]
<img width="815" alt="screen shot 2017-04-11 at 10 11 22 am" src="https://cloud.githubusercontent.com/assets/11826646/24916265/44f4122c-1e9f-11e7-9eeb-499e7f0e43e4.png">

It's a checkbox below the fold, do you see that option?

**Comment by amaliebarras** [comment]
And @Rmcavin can you come to civic hack night tonight at Galvanize? We could catch up about this project there. Otherwise we can set up a google hangout next week.

**Comment by Rmcavin** [comment]
@amaliebarras Sure thing, my boyfriend and I will be there tonight - he is going to help with the project too. I think the issue is that we aren't actually in this GitHub organization yet - I have just been looking for local projects to contribute to and stumbled on this one :) It says online that we need to be invited by an admin (he is bdharris08), then we can join the "all" group. Thanks!

**Comment by amaliebarras** [comment]
hmmm, okay, well in that case let's take a look and sort it out tonight :)

**Comment by Rmcavin** [comment]
Okay sounds good :)

**Comment by amaliebarras** [comment]
hey y'all! have you had the chance to get a repo going for this yet?

**Comment by mateoclarke** [comment]
https://github.com/open-austin/GreenBelts ❓

**Comment by Rmcavin** [comment]
Yep, that's the one! Not much progress yet, but it'll happen lol.

**Comment by werdnanoslen** [comment]
Hi @Rmcavin how's it going? Do you need any help on this project?

**Comment by Rmcavin** [comment]
Hi there! I am currently in the Galvanize Bootcamp and we just got to backend dev these past few days. This has been the missing piece to knowing how to tackle this project, so it should start progressing in the next few days. Sorry for the lack of progress, lots of learning left to do. Once I have a working prototype I will let you guys know where help is needed. Thanks for checking in and sorry again for lack of updates, it is still on my radar especially now that I know what to do :)

**Comment by werdnanoslen** [comment]
No problem, just want to make sure you're still interested and to check if there's anything at all we can help you with. We will actually be at Galvanize for a meetup if you want to hang out with us :)

I'll check back again in a month ✌️

**Comment by tom-montgomery** [comment]
Hey rachel remember me? lol 
Let me know if you want any help on this.

**Comment by Rmcavin** [comment]
Hi Thomas, yes and yes! :) 
So far I got a server running with google maps api in the front end, so the basics exist now. I am researching the best way to store the existing data (right now it's a kml, likely going to do geojson) in a db, and also exploring ways for users to be able to add new geographic and attribute data using the google maps api. There are definitely other things that need to be done too, everything from map styling to working on the dataset. I would love help with whatever aspect of the project interests you, lets chat and I can get you up to speed.
